src/ui/pages/Connections/Connections.test.tsx

```diff
@@ -408,7 +408,7 @@ describe("Connections page from Credentials tab", () => {
     combineMock.mockReturnValue(TabsRoutePath.CREDENTIALS);
   });
 
-  test("It allows to create an Identifier when no Identifiers are available", async () => {
+  test("It allows to create an Identifier when no Identifiers are available (credentials tab)", async () => {
```

Comment:
@sdisalvo-crd @Sotatek-DukeVu This was a duplicate test title, which apparently was causing an issue.

No idea why it is passing on develop, and locally but it was broken on this branch and my other branch. Both branches fixed with this.